PlugMe-ng/plugme-web-ui
# #159192856 User email verification

#### What does this PR do?
Add user account verification functionality
#### Description of Task to be completed?
If a user creates an account, the user's email address should be verified. 
#### How should this be manually tested?
- Goto `https://plugme-client.herokuapp.com/signup`
- Create an account
- Check your email for a verification message
- Click on the link in the message to verify your email address

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
story type: feature
story Id: 159192856
story title: users email address can be verified
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
